src/common/schema.graphql.ts

```diff
@@ -82,6 +82,13 @@ export default gql`
 		SUPER_ADMIN
 	}
 
+	enum EventType {
+		MAIN
+		SPONSOR
+		FOOD
+		HXP
+	}
```

Comment:
maybe more event types - consult with HE team?

Reply:
I'm almost of the persuasion that we should leave this as a string instead of an enum, my reasoning being that the rest of the app is completely generic as far as content goes and only puts a framework in place for rendering that which is configured in a config file. The opposing side of this argument is that most hackathons will have the same types of events, just may refer to them differently.
Would love to hear your thoughts @cktang88

Reply:
i don't have a strong opinion on this